README.md

```diff
@@ -117,19 +140,23 @@ from https://github.com/facebook/watchman
 ## Requirements
 
 * [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
```

Comment:
There are parts of the build that require Java 11 to be installed. This is the new minimum version.

Reply:
Is this true? I have no Java 11 installed, but I can build the Java binding and run the Java tests. It seems that bazel installs java 11 tools automatically.